pkg/build/pipelines/cmake/install.yaml

```diff
@@ -13,4 +13,4 @@ inputs:
 
 pipeline:
   - runs: |
-      DESTDIR="${{targets.contextdir}}" cmake --install ${{inputs.output-dir}}
+      VERBOSE=1 DESTDIR="${{targets.contextdir}}" cmake --install "${{inputs.output-dir}}"
```

Comment:
Should `VERBOSE` be a configurable input that defaults to `0`?

I don't have a strong preference one way or the other.